libs/platform/ebpf_serialize.c

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+// This file contains function implementations for serializing and de-serializing
+// various eBPF structures to/from ebpf_operation*_request/response structures.
+#include "ebpf_serialize.h"
+
+void
+ebpf_map_information_array_free(uint16_t map_count, _In_count_(map_count) const ebpf_map_information_t* map_info)
+{
+    uint16_t map_index;
+
+    for (map_index = 0; map_index < map_count; map_index++) {
+        ebpf_free(map_info[map_index].pin_path);
+    }
+
+    ebpf_free((void*)map_info);
+}
+
+ebpf_result_t
+ebpf_serialize_core_map_information_array(
```

Comment:
Add tests for this code?

Reply:
would execution context unit test be the right place to add this test? There are no platform unit tests. I could add one. Let me know your preference.

Reply:
https://github.com/microsoft/ebpf-for-windows/blob/master/libs/platform/unit/platform_unit_test.cpp

There is a platform unit test already...

Reply:
Added TEST_CASE("serialize_map_test", "[serialize_map_test]").